cmd/2019_7/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/afarbos/aoc/pkg/intcode"
+	"github.com/afarbos/aoc/pkg/mathematic"
+	"github.com/afarbos/aoc/pkg/perm"
+	"github.com/afarbos/aoc/pkg/read"
+	"github.com/afarbos/aoc/pkg/utils"
+)
+
+var flagInput string
+
+func init() {
+	utils.Init(&flagInput)
+}
+
+func compute(instructions []int, inputs []chan int, amplifier, nextAmplifier int, done chan struct{}) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 103 characters, out of limit 80](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)